### PR TITLE
Add Github Actions builds for FreeRTOS+ WinSim Demos

### PR DESCRIPTION
**Pull request by paulbartell**
Add Github Actions builds for FreeRTOS+ Demos

Description
-----------
Update the various FreeRTOS Plus Windows Simulator Demos to target Visual Studio 2019. Build them with github actions.

Test Steps
-----------
Observe that the builds pass.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by paulbartell** [comment]
Split from [PR 381](https://github.com/FreeRTOS/FreeRTOS/pull/831)

File: .github/workflows/freertos_plus_demos.yml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+      - name: Build Cellular MQTT Demo for HL7802
+        working-directory: FreeRTOS-Plus/Demo/FreeRTOS_Cellular_Interface_Windows_Simulator/MQTT_Mutual_Auth_Demo_with_HL7802
+        run: msbuild mqtt_mutual_auth_demo_with_hl7802.sln -t:rebuild -property:Configuration=Debug
+
+      - name: Build Cellular MQTT Demo for SARA-R4
+        working-directory: FreeRTOS-Plus/Demo/FreeRTOS_Cellular_Interface_Windows_Simulator/MQTT_Mutual_Auth_Demo_with_SARA_R4
+        run: msbuild mqtt_mutual_auth_demo_with_sara_r4.sln -t:rebuild -property:Configuration=Debug
+
+  winsim_mqtt:
```
Comment (archigup) [inline .github/workflows/freertos_plus_demos.yml:62]:
```suggestion
  winsim_corelibs:
```

File: manifest.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ dependencies:
       path: "FreeRTOS-Plus/Source/AWS/sigv4"
 
   - name: "coreHTTP"
-    version: "v2.1.0"
+    version: "7fbd7c3"
```
Comment (archigup) [inline manifest.yml:35]: What changes is this for?

Comment (paulbartell) [reply manifest.yml:35]: This includes the lhttp version of coreHTTP. Without it, the coreHTTP demos do not build.

**Review by archigup — APPROVED** [review]

**Review by n9wxu — APPROVED** [review]

**Pull request merged**